tests/test_metadata_frame.py

```
@@ -0,0 +1,153 @@
+    """Test the MetadataSendFrame's ability to set XML data and retrieve it
+
+    Using the setters methods for tag and attributes
+    """
+    frame = MetadataSendFrame(tag='')
+    frame.set_tag(metadata_frame_data.expected.tag)
+    assert frame.get_tag() == metadata_frame_data.expected.tag
+    frame.update(metadata_frame_data.expected.attrs)
+    attrs = {k: v for k, v in frame.items()}
+    assert attrs == metadata_frame_data.expected.attrs
+    xml_data = get_metadata_frame_data(frame)
+    assert normalize_xml(xml_data) == normalize_xml(metadata_frame_data.xml_str)
```

Ruff (0.16.1)

[error] Undefined name `normalize_xml`

(F821)